### PR TITLE
519-update-doc: Update README file

### PR DESCRIPTION
**Pull request by fabricetriboix**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,27 +14,24 @@ and add to your path.
 
 # Usage 
 
-Before usage, you should initialize. It sets up your github integration with your developer environments.
-It requires the following:
-- initialization file: [init-example.yaml](examples/init-example.yaml)
-- your person GitHub token to perform operations on your behalf. See more info [here](#GitHub-Access-Token)
+## Initialisztion
```
Comment (kkonstan) [inline README.md:17]: typo

File: README.md [hunk 2]
```diff
@@ -14,27 +14,24 @@ and add to your path.
...
+
+But first, you will need:
+  - A [GitHub personal token](#GitHub-Access-Token)
+  - To know which GitHub repository is your "environments repository". Please ask your administrator for this.
+
+To initialise `corectl`, run:
 
-To run initialization run:
 ```bash
 corectl config init
```
Comment (kkonstan) [inline README.md:28]: instructions don't work, you get prompted for a file:
```
➜  corectl git:(main) corectl config init
Config not initialised, please run corectl config init.
Select file with configuration for initialization:
> ./ 
```

**Review by kkonstan — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (kkonstan) [reply README.md:17]: Also perhaps we should leave it as is and say it requires either an initialization file, OR an environments repo that contains it.

File: README.md (hunk 2 above)
Comment (fabricetriboix) [reply README.md:28]: @kkonstan I think you are using a different version of `corectl`. I removed the message "Config not initialised, please run corectl config init" when the command is `corectl config init`...

Comment (kkonstan) [reply README.md:28]: of course I am, ignore, works as expected with latest build from main

File: README.md (hunk 1 above)
Comment (fabricetriboix) [reply README.md]: Fixed, thanks for spotting that!

**Review by kkonstan — APPROVED** [review]

**Pull request merged**